DemoApp/package.json

```diff
@@ -13,9 +13,9 @@
     "appcenter-analytics": "^2.0.0",
     "appcenter-crashes": "^2.0.0",
     "appcenter-push": "^2.0.0",
-    "metro": "0.48.1",
+    "metro": "^0.54.1",
     "react": "16.8.3",
-    "react-native": "0.59.2",
+    "react-native": "^0.59.9",
```

Comment:
Our convention is to pin `react` and `react-native` (drop `^` before version number) npm version in test apps. The reasoning is mentioned in a comment (https://github.com/microsoft/appcenter-sdk-react-native/pull/600#discussion_r294987406) from a previous PR.